api/config_entry_exports.go

```diff
@@ -80,3 +82,22 @@ func (e *ExportedServicesConfigEntry) MarshalJSON() ([]byte, error) {
 	}
 	return json.Marshal(source)
 }
+
+type ResolvedExportedService struct {
+	// Service is the name of the service which is exported.
+	Service string
```

Comment:
isn't this missing the json tag?

Reply:
Since this field will always be present, I did not add the `json:",omitempty"` tag.